code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -20,10 +20,16 @@
 			msg += "<span style='font-weight: bold; color: purple;'>You sense this creature is dead.</span>\n"
 		else if(stat || !client)
 			msg += "[span_xenowarning("It doesn't seem responsive.")]\n"
-		if(reagents.get_reagent_amount(/datum/reagent/toxin/xeno_transvitox))
-			msg += "Transvitox: 40% brute/burn injuries received are converted to toxin\n"
+		if(reagents.get_reagent_amount(/datum/reagent/toxin/xeno_neurotoxin))
+			msg += "Neurotoxin: Causes increasingly intense pain and stamina damage over time, incrementing at the 20th and 45th cycle of metabolism\n"
```

Comment:
Talking about metabolism cycles, while accurate, isn't really helpful to someone who's trying to figure out what their abilities do. Could talk about total seconds instead, make it generally more vague, or reword some other way.

Reply:
I'm not sure what the exact seconds are. And aren't ticks usually server dependent? Unsure either way.

Reply:
Life() (which calls chem processing, which in turn increments cycles and metabs) ticks every 2 seconds by default.

Reply:
Got it.